app/controllers/images/licenses_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Images::LicensesController
+module Images
+  class LicensesController < ApplicationController
+    # def access_denied
+    #   flash_error(:permission_denied.t)
+    #   redirect_to(edit_account_preferences_path)
+    # end
+
```

Comment:
```suggestion
```
Remove outdated comment. I was playing with where it would send you if you weren't authorized.